pages/linux/openvpn3.md

```diff
@@ -0,0 +1,24 @@
+# openvpn3
+
+> OpenVPN 3 Linux client.
+> More information: <https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.
+
+- Start new VPN session:
+
+`openvpn3 session-start --config my-vpn-config.conf`
+
+- List established sessions:
+
+`openvpn3 sessions-list`
+
+- Disconnect established session:
```

Comment:
Does this disconnect all connections? If yes how about
```suggestion
- Disconnect all established session:
```
Maybe also on the example above.

Reply:
this does not actually disconnect all sessions. it disconnect a specific session which was started with given configuration. updated the description to hopefully make that more clear